Tree: Repair data in transactions (#12917)
microsoft/FluidFramework: packages/dds/tree/src/repair/README.md

# repair

Abstractions and utilities for managing repair data.

> Repair data corresponds to data that is destroyed when applying edits such as deleted nodes and overwritten values.

Reviewer: ```suggestion
Repair data is data that has been removed from the document, such as deleted nodes and overwritten values.
```
